src/translate/pddl_parser/parsing_functions.py

```diff
@@ -61,6 +61,8 @@ def parse_condition(alist, type_dict, predicate_dict):
 
 def parse_condition_aux(alist, negated, type_dict, predicate_dict):
     """Parse a PDDL condition. The condition is translated into NNF on the fly."""
+    if len(alist) == 0:
```

Comment:
This should be more pythonic:

```
if not alist:
   ...

```
Where did this happen to you?

Reply:
I am not sure I remember. It was either an action with no precondition, or an empty goal statement.

Reply:
To continue in PR: https://github.com/aibasel/downward/pull/60